src/javascript/SelectorTypes/Picker/accordionItems/accordionItems.jsx

```diff
@@ -222,4 +222,17 @@ export const registerAccordionItems = registry => {
             key: 'browse-tree-files'
         }
     });
+
+    setTimeout(() => {
+        const openInJContent = registry.get('action', 'openInJContent');
+        if (openInJContent) {
+            openInJContent.targets.push({id: 'content-editor/pickers/picker-media/header-actions', priority: 0});
+            openInJContent.targets.push({id: 'content-editor/pickers/picker-content-folders/header-actions', priority: 0});
+        }
+
+        const openInPageComposer = registry.get('action', 'pageComposer');
+        if (openInPageComposer) {
+            openInPageComposer.targets.push({id: 'content-editor/pickers/picker-pages/header-actions', priority: 0});
+        }
+    });
```

Comment:
As there is a search mode do we need to register something for it?

Reply:
actually the actions should not appear in search mode